### PR TITLE
Fix details block in Bravia TV

### PR DESCRIPTION
**Pull request by Drafteed**
## Proposed change

https://rc.home-assistant.io/integrations/braviatv/

It looks like the details block doesn't support code blocks.

Bugfix for: https://github.com/home-assistant/core/pull/77329

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by Drafteed** [comment]
Please patch RC if needed.

**Comment by klaasnicolaas** [comment]
What exactly is the purpose of this PR? I see a long list of options in the current documentation and you don't link to a pull request in the core integration?

**Comment by Drafteed** [comment]
@klaasnicolaas This is a bug fix made in my previous PR https://github.com/home-assistant/home-assistant.io/pull/23889.
At the moment it is only merged into `next` and `rc`, not `current`.

**Comment by klaasnicolaas** [comment]
Thank you for explaining. I had to dig a bit to check if your core PR was in the release (it wasn't in the list), but should be fine and your docs PR is now tagged for the release 😉

**Comment by klaasnicolaas** [comment]
image

Maybe it's better not to make this in a list form but just string with commas?

**Comment by Drafteed** [comment]
> but should be fine and your docs PR is now tagged for the release 😉

Thank you! 👍

> Maybe it's better not to make this in a list form but just string with commas?

I thought about it, but it seemed to me that the list is more convenient and readable. Although one line is also good.

**Review by bieniu — DISMISSED** [review]

File: source/_integrations/braviatv.markdown [hunk 1]
```diff
@@ -39,28 +39,7 @@ The integration supports `remote` platform. The remote allows you to send key co
 The commands that can be sent to the TV depends on the model of your TV. To display a list of supported commands for your TV, call the service `remote.send_command` with non-valid command (e.g. `Test`). A list of available commands will be displayed in [Home Assistant System Logs](https://my.home-assistant.io/redirect/logs).
 
 {% details "Some commonly used commands" %}
```
Comment (frenck) [inline source/_integrations/braviatv.markdown:41]: The change needed here, is adding a new-line before and after the liquid syntax (`{% ... %}`, allowing the parser to switch context between liquid and Markdown.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/braviatv.markdown [hunk 2]
```diff
@@ -39,7 +39,9 @@ The integration supports `remote` platform. The remote allows you to send key co
...
+
 Up, Down, Left, Right, Confirm, Return, Home, Exit, Rewind, Forward, ActionMenu, SyncMenu, Num0, Num1, Num2, Num3, Num4, Num5, Num6, Num7, Num8, Num9
```
Comment (frenck) [inline source/_integrations/braviatv.markdown]: Please restore the previous list. A alphabetically list is easier to scan than single line with tons of options.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @Drafteed 👍

**Pull request merged**